pyproject.toml

```
@@ -145,71 +120,223 @@ saritasa-invocations = "^0.10.0"
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.isort]
-line_length = 79
-multi_line_output = 3
-known_django = "django"
-known_drf = "rest_framework"
-known_apps = "import_export_extensions"
-skip = [
-    "migrations",
-    "_tmp,src",
-    "scripts",
-    ".venv",
-    "node_modules",
-    "config",
-]
-sections = [
-    "FUTURE",
-    "STDLIB",
-    "DJANGO",
-    "DRF",
-    "THIRDPARTY",
-    "FIRSTPARTY",
-    "APPS",
-    "LOCALFOLDER",
-]
-include_trailing_comma = true
-default_section = "THIRDPARTY"
+[tool.ruff]
+line-length = 79
+indent-width = 4
+target-version = "py311"
```

Since we still support `python 3.10`
```suggestion
target-version = "py310"
```